### DISCUSSION TITLE
Display a total cost of items in the shopping list

### DISCUSSION THREAD

**Discussion by Mills79** [hassio-addons/addon-grocy#124] [Q&A] [answered]
I think it would be nice for their to be a display of the total cost of the items in the shopping list so that you would have a rough idea of how much you are going to spend on the shopping before you even get to the shop.

**Comment by frenck** [comment, marked as answer]
Hi there @Mills79 👋 

This place is about the add-on. Add-ons wrap existing software; in this case Grocy. That means we pack Grocy in a way that is useable with Home Assistant as an add-on. 

However, we don't develop the Grocy software. Please submit these type of feature requests with Grocy themselves 👍
